src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ExplicitInitializationCheck.java

```diff
@@ -55,6 +55,39 @@
  * &lt;module name=&quot;ExplicitInitialization&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   private int a = 0; // violation
```

Comment:
please change field names to make them distinct and make example compilable.

Reply:
@ImmortalRabbit new field naming is not good. Please use more common, like intField, objField, etc.